schema/tables/tcc_access.yaml

```diff
@@ -0,0 +1,46 @@
+name: tcc_access
+platforms:
+  - darwin
+description: Information about macOS TCC database contents, for determining permissions granted to applications.
+columns:
+  - name: source
+    type: text
+    required: true
+    description: Either "user" or "system".
+  - name: service
+    type: text
+    required: true
+    description: The name of the TCC service.
+  - name: client
+    type: text
+    required: true
+    description: The bundle identifier or absolute path to the program using the TCC service.
+  - name: client_type
+    type: integer
+    required: true
+    description: Indicates whether client is a bundle identifier (0) or absolute path (1). 
+  - name: auth_value
+    type: integer
+    required: true
+    description: 'Indicates whether the access is: denied (0), unknown (1), allowed (2), or limited (3).'
+  - name: auth_reason
+    type: integer
+    required: true
+    description: TODO
+  - name: last_modified
+    type: bigint
+    required: true
```

Comment:
@rachaelshaw can you please clarify the meaning of these `required` fields? It wouldn't seem to make sense to require the user to specify a `last_modified` value in their query.